### DISCUSSION TITLE
Azure AD Proxy - policzenie kosztów.

### DISCUSSION THREAD

**Discussion by phaate** [AzureCommunityPL/group#288] [Pomoc]
Hej,

Mam małą zagwozdkę z licencjonowaniem i kosztami dla Azure AD Proxy, nigdy z tego nie korzystałem bo zawsze zaporą było P1 (5$ miesiecznie) ale popełniłem teraz małe MVP i... zapomniałem wrzucić userom licencję na AAD Premium a i tak mogli korzystać z aplikacji postawionej na on-premie.

Jedyne czego potrzebuje to "schować" appkę na on-premie za Azure AD i jako MFA - MS Authenticator.

I teraz pytanie - czy do tej funkcjonalności nadal jest wymagana licencja AAD P1?

**Comment by eklime** [comment]
Technicznie, potrzebujesz jedną aktywną licencję P1 w tenancie aby App Proxy było "aktywne".

**Reply by phaate** [reply]
@eklime Jeżeli dobrze rozumiem - technicznie muszę mieć minimum jedną licencję P1 do wystawienia Proxy ale "prawnie" podobnie jak z CALami -> Każdemu userowi muszę nadać tą licencję jak łączy się z aplikacją?

... Czy na jednej licencji mogę opędzić całą firmę biorąc pod uwagę wymagania?
